packages/dashboard/src/PanelManager.ts

```diff
@@ -162,8 +162,22 @@ class PanelManager {
     return undefined;
   }
 
-  getLastUsedPanelOfType(type: ComponentType): PanelComponent | undefined {
-    return this.getLastUsedPanel(panel => panel instanceof type);
+  getLastUsedPanelOfType(type: PanelComponentType): PanelComponent | undefined {
+    return this.getLastUsedPanelOfTypes([type]);
+  }
+
+  getLastUsedPanelOfTypes(
+    types: PanelComponentType[]
+  ): PanelComponent | undefined {
+    return this.getLastUsedPanel(panel =>
+      types.some(
+        type =>
+          panel instanceof type ||
+          (isWrappedComponent(type) &&
```

Comment:
Can we get away with a simple `panel => panel instanceof type` check here? just pass unwrapped components to `getLastUsedPanelOfTypes` and skip the `isWrappedComponent` check? I didn't think of this solution when I added this on the enterprise side, then I saw you had it simplified in DHC. Then I guess remove `| WrappedComponentType<C, P>` from the `PanelComponentType` definition too.

Reply:
I've fixed this, but for posterity, we still need the WrappedComponentType for registerComponent anyway, so may as well make it work right!